README.md

```diff
@@ -1,5 +1,7 @@
 Megatron
 ========
+# NOTE: THIS REPO HAS BEEN ARCHIVED AS OF 2002-08-10
```

Comment:
I think you mean 2022. But I don't think we need a date really.

Reply:
fix0red